packages/ui/src/components/va-modal/plugin/index.ts

```diff
@@ -0,0 +1,22 @@
+import { App } from 'vue'
+import { defineVuesticPlugin, defineGlobalProperty } from '../../../vuestic-plugin/utils'
+import { createModalInstance } from '../modal'
+import { ModalOptions } from '../types'
+
+const createVaModalPlugin = (app: App) => ({
+  init (options: string | ModalOptions) {
+    return createModalInstance(options, app?._context)
```

Comment:
Not app?.appContext? Think we shouldn't use private properties.

Reply:
we don't have access to `appContext` here, only `_context`

Reply:
Opened new issue #1853